### PR TITLE
Divide i-seg tests from detection tests

### PR DESCRIPTION
**Pull request by sungmanc**
### Summary

Divided Instance segmentation tests from detection tests

### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added e2e tests for validation.
- [ ] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md) in develop).​
- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) in develop).
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

### License

- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/openvinotoolkit/training_extensions/blob/develop/LICENSE) that covers the project.
      Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

### CODE REVIEW

**Review by yunchu — DISMISSED** [review]

File: tox.ini [hunk 1]
```diff
@@ -15,7 +15,8 @@ test_dir =
     ano: cli/anomaly
     cls: cli/classification
     det: cli/detection
-    seg: cli/segmentation
+    i_seg: cli/instance_segmentation
+    s_seg: cli/semantic_segmentation
```
Comment (goodsong81) [inline tox.ini]: We don't need this separation. This cls/det/seg means just `extra` dependency in pip install.

**Review by goodsong81 — CHANGES_REQUESTED** [review]
Could you revert i_seg/s_seg in tox, and just run differently in GH workflow?

**Review by supersoob — DISMISSED** [review]

File: tox.ini (hunk 1 above)
Comment (sungmanc) [reply tox.ini]: Done

**Review by goodsong81 — DISMISSED** [review]
LGTM

**Review by eunwoosh — DISMISSED** [review]

**Review by eunwoosh — APPROVED** [review]

**Review by eunwoosh — APPROVED** [review]

**Review by goodsong81 — APPROVED** [review]

**Pull request merged**